kolibri/plugins/management/assets/src/vue/modal/index.vue

```diff
@@ -120,6 +113,12 @@
       focusModal() {
         this.$els.modal.focus();
       },
+      focusElementTest(event) {
+        // FocusOut happens when the element is about to be blurred
+        if (!this.$els.modal.contains(event.relatedTarget)) {
```

Comment:
I'm surprised we're using focusout and not focusin. I would have guessed that if the dom element about-to-be-blurred passes this test, we've already tabbed too far?